lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""This library facilitates the prometheus_remote_write interface.
+
+For the provider charm no communication is necesssary. For the consumer charm,
+instantite a PrometheusRemoteWriteConsumer in the __init__ method of your charm
+class:
+
+```
+from charms.prometheus_k8s.v0.prometheus_remote_write import PrometheusRemoteWriteConsumer
+
+def __init__(self, *args):
+    ...
+    self.remote_write = PrometheusRemoteWriteConsumer(self, "remote-write")
+    ...
+```
+
+Then access the endpoints with:
```

Comment:
It is not clear why a charm instantiating the `PrometheusRemoteWriteConsumer` would want to access the remote write endpoints or the config. I wonder if some clarification here perhaps with an example will be useful. For instance "use the config retuned by `config()` to set the ...."

Reply:
I am waiting to fix the library docs until it is decided what the library will look like. Changes will be made here for sure